db/migrate_core_cards/20141230204340_uri_codename.rb

```diff
@@ -0,0 +1,11 @@
+# -*- encoding : utf-8 -*-
+
+class UriCodename < Wagn::Migration
+  def up
+    contentedly do
+      puts "adding new codename card for URI"
+      codename = 'URI'
+      Card.create! :type_id=>Card::CardtypeID, :name=>codename, :codename=>codename.to_name.key
```

Comment:
the general solution isn't, true, but we've got to get a fix of some sort
in for this before we release it.  migration issues are a huge headache.

On Fri, Jan 2, 2015 at 8:01 PM, Gerry Gleason <notifications@github.com>
wrote:

> In db/migrate_core_cards/20141230204340_uri_codename.rb
> <https://github.com/wagn/wagn/pull/281#discussion-diff-22429000>:
>
> > @@ -0,0 +1,11 @@
> > +# -*- encoding : utf-8 -*-
> > +
> > +class UriCodename < Wagn::Migration
> > +  def up
> > +    contentedly do
> > +      puts "adding new codename card for URI"
> > +      codename = 'URI'
> > +      Card.create! :type_id=>Card::CardtypeID, :name=>codename, :codename=>codename.to_name.key
>
> In my use of codename, this is the codename, just not in 'key' form. It is
> also the initial name, but it may not be. 'autoname' doesn't seem the right
> name for this feature, maybe auto_new_name ? Doesn't seem that related to
> this ticket in any case.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/wagn/wagn/pull/281/files#r22429000>.
>



-- 
Ethan McCutchen
One of the Wagneers, Wagn.org

Wagn. How pioneers roll.

s: ethan.mccutchen
t: @intogreater